app/javascript/packages/verify-flow/steps/personal-key/personal-key-step.tsx

```diff
@@ -45,15 +46,27 @@ function PersonalKeyStep({ value }: PersonalKeyStepProps) {
       <DownloadButton
         content={personalKey}
         fileName="personal_key.txt"
+        onClick={() => {
+          trackEvent('IdV: download personal key');
+        }}
```

Comment:
For compactness, could we collapse these click handlers to be inline?

```suggestion
        onClick={() => trackEvent('IdV: download personal key')}
```